app/controllers/charges_controller.rb

```diff
@@ -1,6 +1,11 @@
 class ChargesController < ApplicationController
   def new
     @project = Project.find(params[:project_id])
+    if @project.developers.any? {|dev| dev.status == 'unavailable'}
```

Comment:
Space between { and | missing.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.